albumentations/augmentations/transforms.py

```diff
@@ -3177,3 +3178,146 @@ def get_transform_init_args_names(self):
+            ratio (float): the ratio of the mask holes to the unit_size (same for horizontal and vertical directions).
+                Must be between 0 and 1. Default: 0.5.
+            unit_size_min (int): minimum size of the grid unit. Must be between 2 and the image shorter edge.
+                If 'None', holes_number_x and holes_number_y are used to setup the grid. Default: `None`.
+            unit_size_max (int): maximum size of the grid unit. Must be between 2 and the image shorter edge.
+                If 'None', holes_number_x and holes_number_y are used to setup the grid. Default: `None`.
+            holes_number_x (int): the number of grid units in x direction. Must be between 1 and image width//2.
+                If 'None', grid unit width is set as image_width//10. Default: `None`.
+            holes_number_y (int): the number of grid units in y direction. Must be between 1 and image height//2.
+                If `None`, grid unit height is set equal to the grid unit width or image height, whatever is smaller.
+            shift_x (int): offsets of the grid start in x direction from (0,0) coordinate.
+                Clipped between 0 and grid unit_width - hole_width. Default: 0.
+            shift_y (int): offsets of the grid start in y direction from (0,0) coordinate.
+                Clipped between 0 and grid unit height - hole_height. Default: 0.
+            random_offset (boolean): weather to offset the grid randomly between 0 and grid unit size - hole size
+                If 'True', entered shift_x, shift_y are ignored and set randomly. Default: `False`.
+            fill_value (int): value for the dropped pixels. Default = 0
+            mask_fill_value (int): value for the dropped pixels in mask.
+                If `None`, tranformation is not applied to the mask. Default: `None`.
+        Targets:
+            image, mask
+        Image types:
+            uint8, float32
+        References:
+            https://arxiv.org/pdf/2001.04086.pdf7
+    """
+
+    def __init__(
+        self,
+        ratio: float = 0.5,
+        unit_size_min: int = None,
+        unit_size_max: int = None,
+        holes_number_x: int = None,
+        holes_number_y: int = None,
+        shift_x: int = 0,
+        shift_y: int = 0,
+        random_offset: bool = False,
+        fill_value: int = 0,
+        mask_fill_value: int = None,
+        always_apply: bool = False,
+        p: float = 0.5,
+    ):
+        super(GridDropout, self).__init__(always_apply, p)
+        self.ratio = ratio
+        self.unit_size_min = unit_size_min
+        self.unit_size_max = unit_size_max
+        self.holes_number_x = holes_number_x
+        self.holes_number_y = holes_number_y
+        self.shift_x = shift_x
+        self.shift_y = shift_y
+        self.random_offset = random_offset
+        self.fill_value = fill_value
+        self.mask_fill_value = mask_fill_value
+        assert 0 < self.ratio < 1, "ratio must be between 0 and 1."
```

Comment:
I am not sure, but maybe there need conditions looks like: `0 < self.ratio <= 1`